dev-docs/bidders/adplus.md

```diff
@@ -8,7 +8,7 @@ pbjs: true
 ---
 ### Note:
 
-The AdPlus Bidding adapter requires setup before beginning. Please contact us at adplus.destek@yaani.com.tr
+The AdPlus Bidding adapter requires setup before beginning. Please contact us at <adplusdestek@turkcell.com.tr>.
```

Comment:
If you want this to be a proper HTML email tag, it would be 

```
<a href = "mailto: adplusdestek@turkcell.com.tr">adplusdestek@turkcell.com.tr</a>
```

Reply:
Just commenting here to bump it. Can you verify the email address and how you formatted it in the file. Thanks!

Reply:
actually, I just found out that jekyll does the right thing with <email>. Merging this. Sorry for the delay.